engine/src/main/java/org/terasology/world/block/family/AttachedToSurfaceFamily.java

```diff
@@ -68,7 +68,7 @@ public Block getArchetypeBlock() {
     public Block getBlockFor(BlockUri blockUri) {
         if (getURI().equals(blockUri.getFamilyUri())) {
             try {
-                Side side = Side.valueOf(blockUri.getIdentifier().toString().toUpperCase(Locale.ENGLISH));
+                Side side = Side.valueOf(blockUri.blockName.toString().toUpperCase(Locale.ENGLISH));
```

Comment:
Please don't use other classes members directly. This shouldn't even be possible. A legacy issue I assume.
Afaik, simple getters are compiled to the same bytecode as using the members directly, so their performance is identical.